code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -45,7 +45,7 @@
 	else
 		var/datum/gas_mixture/heating = air_contents.remove_ratio(exposed_volume/air_contents.volume)
 		heating.temperature = exposed_temperature
-		heating.react()
+		heating.react(src)
```

Comment:
this is just for consistency, shouldn't have any affect on gameplay but i'll test it since i learned my lesson the last time